tests/components/rfxtrx/test_cover.py

```diff
@@ -55,20 +55,25 @@ async def test_one_cover(hass, rfxtrx):
     )
     await hass.async_block_till_done()
 
-    hass.data[rfxtrx_core.DATA_RFXOBJECT] = rfxtrxmod.Core(
-        "", transport_protocol=rfxtrxmod.DummyTransport
+    assert len(hass.states.async_all()) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)